src/components/accordion/Accordions.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Accordion, { AccordionProps } from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+
+import { styles } from '~/components/accordion/Accordion.styles'
+
+import { AccordionItem, AccordionSx } from '~/types'
+import { Variant } from '@mui/material/styles/createTypography'
```

Comment:
in some files you use variantEnum from types. but here this from mui, you should stick to one approach

Reply:
i used Variant as type from MUI just to type some field and VariantEnum as constants for adding values to typography, do you see it as a problem to use MUI types for typing input values in MUI component such as variant for Typography?

Reply:
no, it is not problem, but if we have enum for typography variants we can use it not only for adding value but also for type checking